client/boot/locale.js

```diff
@@ -66,7 +66,11 @@ export const setupLocale = ( currentUser, reduxStore ) => {
 		reduxStore.dispatch( setLocale( currentUser.localeSlug, currentUser.localeVariant ) );
 	}
 
-	if ( config.isEnabled( 'use-translation-chunks' ) && '__requireChunkCallback__' in window ) {
+	const useTranslationChunks =
+		config.isEnabled( 'use-translation-chunks' ) ||
+		/[?,&]useTranslationChunks(=.*)?$/.test( document.location.search );
```

Comment:
Maybe use our Calypso's [`getUrlParts`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/url/url-parts.ts#L79)?
```suggestion
		config.isEnabled( 'use-translation-chunks' ) ||
		getUrlParts(document.location).searchParams.useTranslationChunks
```

Reply:
I wasn't aware of this, thanks for the suggestion!